ts/features/messages/components/MessageAttachment/MessageAttachment.tsx

```diff
@@ -44,65 +48,59 @@ const MessageAttachmentFooter = ({
   name
 }: MessageAttachmentFooterProps) =>
   isIos ? (
-    <FooterWithButtons
-      type={"SingleButton"}
-      primary={{
-        type: "Solid",
-        buttonProps: {
+    <FooterActions
+      actions={{
+        type: "SingleButton",
+        primary: {
+          label: I18n.t("messagePDFPreview.singleBtn"),
           accessibilityLabel: I18n.t(
             "messagePDFPreview.singleBtnAccessibility"
           ),
           onPress: () => {
             onShare(isPN, attachmentCategory);
             ReactNativeBlobUtil.ios.presentOptionsMenu(downloadPath);
-          },
-          label: I18n.t("messagePDFPreview.singleBtn")
+          }
         }
       }}
     />
   ) : (
-    <FooterWithButtons
-      type={"TwoButtonsInlineHalf"}
-      secondary={{
-        type: "Solid",
-        buttonProps: {
-          accessibilityLabel: I18n.t("messagePDFPreview.shareAccessibility"),
-          onPress: () => {
-            onShare(isPN, attachmentCategory);
-            share(`file://${downloadPath}`, undefined, false)().catch(_ => {
-              IOToast.show(I18n.t("messagePDFPreview.errors.sharing"));
-            });
-          },
-          label: I18n.t("messagePDFPreview.share")
+    <FooterActionsInline
+      startAction={{
```

Comment:
Same mapping issue here.
In this case, did you swap them on purpose, or was it a mistake?

Reply:
In this particular case there are two competing buttons (both are `Solid`), but `FooterActionsInline` is is designed to display `Outline` and `Solid` inline, respectively. Actual (above) versus proposed (below):

<img src="https://github.com/user-attachments/assets/3a18902b-cfa0-42de-ba09-ece4711c8749" width="315" />

(cc @Vangaorth)